eng/testing/debug-dump-template.md

```diff
@@ -111,13 +111,15 @@ lldb --core <lin-path-to-dump> %LOUTDIR%/shared/Microsoft.NETCore.App/6.0.0/dotn
 ```
 
 Within lldb:
-```
-setclrpath %LOUTDIR%/shared/Microsoft.NETCore.App/6.0.0
+```gdb
+setclrpath /home/<your username>/helix_payload/%WORKITEM%/shared/Microsoft.NETCore.App/6.0.0
```

Comment:
Now that you're here, would you mind updating 6.0.0 to 7.0.0 please? (Probably should be a parameter, but given they already have to edit this it's not a big deal).

Reply:
Also, what do you think of these:

- Would caps bring more attention to the message
- also, we could merge with the core command: ``-c setclrpath `pwd`/shared/Microsoft.NETCore.App/7.0.0``

Reply:
Changed 6 to 7 with caps for `<YOUR USERNAME>`.

I couldn't make command line arg work:

```sh
$ dotnet-dump analyze -c \
  "setclrpath ~/helix_payload/shared/Microsoft.NETCore.App/7.0.0" \
  "setsymbolserver -directory ~/helix_payload/System.Text.Json.Tests/shared/Microsoft.NETCore.App/7.0.0" \
  ~/helix_payload/System.Text.Json.Tests/workitems/System.Text.Json.Tests/core.1001.23 

Required argument missing for command: analyze

analyze:
  Starts an interactive shell with debugging commands to explore a dump

Usage:
  dotnet-dump analyze [options] <dump_path>

Arguments:
  <dump_path>    Name of the dump file to analyze.

Options:
  -c, --command <command>    Run the command on start. [default: System.String[]]
  -?, -h, --help             Show help and usage information
```

Reply:
Ah, with different order, it worked:

```sh
$ dotnet-dump analyze ~/helix_payload/System.Text.Json.Tests/workitems/System.Text.Json.Tests/core.1001.23 \
  -c "setclrpath $(pwd)/helix_payload/System.Text.Json.Tests/shared/Microsoft.NETCore.App/7.0.0" \
  "setsymbolserver -directory $(pwd)/helix_payload/System.Text.Json.Tests/shared/Microsoft.NETCore.App/7.0.0"
```

I'll update.

Reply:
Done, `lldb` uses `-o` (small `o` is for option to be set after the target is set, capital is before) and it should repeat for each option. `dotnet-dump` uses `-c` which accepts (quoted) space separated (string[]) options. `-c` in `analyze` command has to come after core file.

to keep both worlds happy, I replaced `~` with `$HOME` in generator script.